packages/editor/src/client/routes/articleEditor.tsx

```diff
@@ -192,6 +192,24 @@ export function ArticleEditor({id}: ArticleEditorProps) {
     }
   }
 
+  // Reads title and lead from the first block and saves them in variables
+  function syncBlocksWithMeta() {
+    const titleBlock = blocks.map(block => {
+      if (block.type === 'title') {
+        return block.value
+      } else {
+        return false
+      }
+    })
+
+    const syncedTitle = titleBlock[0] ? titleBlock[0].title : ''
+    const syncedLead = titleBlock[0] ? titleBlock[0].lead : ''
+    const title = metadata.title !== '' ? metadata.title : syncedTitle
```

Comment:
check first if metadata is empty. Because if not you don't have to search for the right block.